packages/databricks-vscode/src/bundle/models/BundleValidateModel.ts

```diff
@@ -25,9 +26,14 @@ export class BundleValidateModel extends BaseModelWithStateCache<BundleValidateS
     ) {
         super();
         this.disposables.push(
-            this.bundleWatcher.onDidChange(async () => {
-                await this.stateCache.refresh();
-            })
+            this.bundleWatcher.onDidChange(
+                withOnErrorHandler(
+                    async () => {
+                        await this.stateCache.refresh();
+                    },
+                    {log: true, throw: false}
```

Comment:
Aren't those the defaults?

Reply:
Yes, I just felt it is easier to keep track of what thing does what and what doesn't this way.